pkg/virtctl/guestfs/guestfs.go

```diff
@@ -605,7 +605,7 @@ func createAttacher(client *K8sClient, p *corev1.Pod, command string) error {
 	resChan := make(chan error)
 
 	go func() {
-		resChan <- exec.Stream(remotecommand.StreamOptions{
+		resChan <- exec.StreamWithContext(context.TODO(), remotecommand.StreamOptions{
```

Comment:
This commit does more than it says, please separate this into its own contained commit.

Reply:
updated the description to include this change as well - its replacing 2 additional function calls. I could split this PR into 2 - one for pkg and one for function call deprecations?